src/autopas/tuning/AutoTuner.h

```diff
@@ -281,12 +281,17 @@ class AutoTuner {
   /**
    * Number of iterations between two tuning phases.
    */
-  size_t _tuningInterval;
+  std::size_t _tuningInterval;
```

Comment:
You are correct that using `std::size_t` is cleaner than `size_t`, but I fear mixing them in the codebase might lead to confusion (especially among students). We should probably go through the whole code at some point, replace it, and add a custom CI check. Do NOT do this in this PR, but maybe create an issue for it.

Reply:
Yeah, you are right, especially since there is a normal `size_t`  variable just above it.
But just fyi, our codebase is already mixed.